src/add_remove_update.js

```diff
@@ -56,7 +56,8 @@ export class Store {
     const checkbox = el;
     const taskList = document.querySelector('.tasks');
     const tasks = Store.getTasks();
-    const nodes = Array.prototype.slice.call(taskList.children);
+    const nodes =Array.prototype.slice.call(taskList.children);;
+    console.log(nodes)
     for (let i = 0; i < tasks.length; i += 1) {
```

Comment:
you can use  (.forEach(), .map(), .filter(), .reduce() and .find()) functions instead of for loop for more clean code .